data/graphics.s

```diff
@@ -16086,10 +16086,28 @@ gUnknown_8E97EC4:: @ 8E97EC4
 	.incbin "baserom.gba", 0xE97EC4, 0x120
 
 gUnknown_8E97FE4:: @ 8E97FE4
-	.incbin "baserom.gba", 0xE97FE4, 0x40 @UNKNOWN
+	.incbin "baserom.gba", 0xE97FE4, 0x20 @UNKNOWN
+
+gUnknown_8E98004:: @ 8E98004
+	.incbin "graphics/interface/naming_screen_8E98004.gbapal"
 
 gUnknown_8E98024:: @ 8E98024
-	.incbin "baserom.gba", 0xE98024, 0xC0 @UNKNOWN
+	.incbin "graphics/interface/naming_screen_8E98024.gbapal"
+
+gUnknown_8E98044:: @ 8E98044
+	.incbin "graphics/interface/naming_screen_8E98044.gbapal"
+
+gUnknown_8E98064:: @ 8E98064
+	.incbin "graphics/interface/naming_screen_8E98064.gbapal"
+
+gUnknown_8E98084:: @ 8E98084
+	.incbin "graphics/interface/naming_screen_8E98084.gbapal"
+
+gUnknown_8E980A4:: @ 8E980A4
+	.incbin "graphics/interface/naming_screen_8E980A4.gbapal"
+
+gUnknown_8E980C4:: @ 8E980C4
+	.incbin "graphics/interface/naming_screen_8E980C4.gbapal"
 
 gUnknown_8E980E4:: @ 8E980E4
 	.incbin "baserom.gba", 0xE980E4, 0x1D8
```

Comment:
Is it possible to resolve these incbins?

Reply:
Would rather hold off on that until other objects have been dumped completely.